UI/Web/src/app/series-detail/series-detail.component.ts

```diff
@@ -126,7 +126,7 @@ export class SeriesDetailComponent implements OnInit, OnDestroy, AfterContentChe
   /**
    * Track by function for Chapter to tell when to refresh card data
    */
-  trackByChapterIdentity = (index: number, item: Chapter) => `${item.title}_${item.number}_${item.pagesRead}`;
+  trackByChapterIdentity = (index: number, item: Chapter) => `${item.title}_${item.number}_${item.volumeId}_${item.pagesRead}`;
```

Comment:
This is needed by the scroller to properly track issues. Without it, the cards of issues sharing a same number randomly rearrange themselves while scrolling.